app/controllers/logs_controller.rb

```diff
@@ -44,7 +44,7 @@ def tardy
   end
 
   def index(logs=nil, header='Entire Log')
-    filter = filter.nil? ? '' : " AND #{filter}"
+    filter.nil? ? '' : " AND #{filter}"
```

Comment:
Hmmm does this do something I don't understand? Seems like the assignment was lost?